fast_carpenter/summary/binned_dataframe.py

```diff
@@ -3,6 +3,7 @@
 """
 import os
 import pandas as pd
+import numpy as np
```

Comment:
I think this would now fail the flake8 test because `np` isn't used anywhere.  I think it's ok you don't pick up the .travis-ci.yml file.